cmd/okctl/version.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/oslokommune/okctl/pkg/okctl"
+	"github.com/spf13/cobra"
+)
+
+// nolint: globalvar
+// The current git commit at build-time with goreleaser
+var GitCommit string
+
+// nolint: globalvar
+// The current version of OkCLI
```

Comment:
It's okCTL not okCLI

Reply:
Actually that is also wrong, I've checked the canonical source and the correct way is just "okctl"

Reply:
Sorry about the confusion, fixed now!